ReadmeSrc/Readme.js

```diff
@@ -0,0 +1,116 @@
+require('dotenv').load()
+const axios = require('axios')
+const moment = require('moment')
+const merge = require('lodash/fp/merge')
+const table = require('markdown-table')
+const delay = require('delay')
+
+const {getRows, headers} = require('./Rows')
+
+const data = require('../webpage/src/data.json')
```

Comment:
It would make sense to save fetched data back to `/webpage/src/data.json` this way it can be reused by webpage.

Reply:
Yes. I see this as a separate PR though. I would do something more with the new data and the web app, sorting, filtering, more data.

Reply:
Yes I do not ask you to change web application. I'm asking to write this data to JSON file.

```js
const writeOut = (data, path) =>
  new Promise((resolve, reject) =>
    fs.writeFile(path, JSON.stringify(data, null, 2), "utf-8", err => {
      if (err) reject(err);
      else resolve(data);
    })
  );
await writeOut(data, '../webpage/src/data.json')
```